content/docs/engine-dev/assets/filetypes/texture-support.md

```diff
@@ -0,0 +1,113 @@
+
+FBX and other scene files also reference source image assets via path, and there's nothing we can do about that. It's a known limitation.
+
+However, the gotcha here is on cache consistency: Right now, without a job dependency, when renaming an image like in the previous section, the FBX file referencing the image will not reprocess. The product asset azmaterial of the FBX file references the product asset via asset ID, but the source asset FBX referenced via path. So now with the rename, the stale azmaterial product asset of the FBX file has a valid asset ID referencing the relocated asset. This means that an asset relocation of an image file may appear to work immediately: rename `CarBody.png` to `CarBody1.png`, and the `Car.azmaterial` product asset of `Car.fbx` appears to render correctly with the asset ID based reference to `CarBody1.streamingimage`. Then, later on someone makes a change to cause `Car.fbx` to reprocess and now the image can't be found and there is an error or warning processing `Car.fbx`. If a user then panics and tries to roll back this change to `Car.fbx`, it will still appear broken, because they had previously been in an unstable state.
+#### Why is this concern?
+
+Cache consistency is very important. Situations like this can break user trust in asset processing, create situations where projects work differently on different team members machines, and the time between causing the issue (renaming the texture) and discovering the bug (the reference is broken, the scene builder can't find the texture and the azmaterial product asset no longer has a valid texture) may take a long time. If everyone on the team is working with iterative caches day to day, and someone finally clears their cache or sets up a new project several weeks after the rename, that's a lot of other changes that could have caused the problem.
+#### What can we do to address this?
+
+* Update scene files to emit job dependencies on referenced source asset image files.
+* Make sure job dependencies are using asset IDs instead of paths before being recorded into the asset database, so that renaming an image file will cause the dependency job for the new name to match the old dependency, and cause the scene files to reprocess.
+
+This will mean that renaming an image referenced by an FBX file will cause that FBX file to reprocess, due to the job dependency. This will maintain cache consistency for a team, and make it easy to quickly find issues when an image was referenced by an FBX file, so the team can go upstream to adjust the Blender, Maya, or other working asset to point at the new image, and re-export the FBX with the new path.
+### Scene file source assets have path based references to images

```

Comment:
```suggestion
This will mean that renaming an image referenced by an FBX file will cause that FBX file to reprocess, due to the job dependency. This will maintain cache consistency for a team, and make it easy to quickly find issues when an image was referenced by an FBX file, so the team can go upstream to adjust the Blender, Maya, or other working asset to point at the new image, and re-export the FBX with the new path.

### Scene file source assets have path based references to images

```